### DISCUSSION TITLE
android tv app and apple tv app

### DISCUSSION THREAD

**Discussion by OneOfOneGH** [f1multiviewer/issue-tracker#473] [Feature requests]
as stated. your app is great on pc but it would be epic on android tv [nvidia shield]. I watch in my theater on an 83 inch oled with 5.1.2 surround sound on the shield using the F1.tv app. since your app is much better it would be amazing to experience on andriod tv.

**Comment by LapsTimeOFF** [comment]
Duplicate of #546
